packages/edit-widgets/src/components/notices/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { filter } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { SnackbarList } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+
+function Notices() {
```

Comment:
This made me wonder if we want to move the `EditorNotices` component from the `editor` package to the `notices` package as we'd have to rewrite such component for each consumer.

Reply:
Yes, I guess it would be good to avoid this code repetition and just expose the component as part of the notices package.